### PR TITLE
Overhaul the auth cookie documentation

### PR DESCRIPTION
**Pull request by supersven**
- Fix wrong statements
- Add configuration example

## Checklist

 - [X] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [X] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

File: docs/src/understand/api-client-perspective/authentication.rst [hunk 1]
```diff
@@ -62,12 +63,34 @@ login <#login-sms>`__. The response to a successful login contains an
...
+session cookies, persistent cookies are replaced by persistent cookies.
+
+To prevent performance issues and malicious usages of the API, there is a
+throttling mechanism in place. When the maximum number of cookies of one type
+are issued, it's checked that login calls don't happen too frequently (too
+quickly after one another.)
+
+In case of throttling no cookie gets issued. The error response (`HTTP status
+code 429 <https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/429>`_) has
+an ``Retry-After`` header which specifies the time to wait before accepting the
```
Comment (pcapriotti) [inline docs/src/understand/api-client-perspective/authentication.rst:85]: an → a

File: docs/src/understand/api-client-perspective/authentication.rst [hunk 2]
```diff
@@ -62,12 +63,34 @@ login <#login-sms>`__. The response to a successful login contains an
...
+next request in Seconds.
+
+Being throttled is a clear indicator of incorrect API usage. There is no need to
+login many times in a row on the same device. Instead, the cookie should be
+re-used.
+
+The corresponding backend configuration settings are described in:
+:ref:`auth-cookie-config`
```
Comment (pcapriotti) [inline docs/src/understand/api-client-perspective/authentication.rst:93]: Missing full stop.

**Review by pcapriotti — APPROVED** [review]
Looks good. Minor comments below.

**Comment by supersven** [comment]
> Looks good. Minor comments below.

@pcapriotti , Thanks a lot for grammar and spell checking!

**Pull request merged**